src/css/main.scss

```diff
@@ -478,4 +488,236 @@ button {
+      margin-bottom: 1.5rem;
+
+      @include media('lg-up') {
+        margin-bottom: 4.625rem;
+      }
+    }
+
+    // headings
+    h1 {
+      @extend .text-subheader-alt;
+      margin-bottom: 1.5rem;
+      color: color('black');
+
+      @include media('md-up') {
+        margin-bottom: 4rem;
+      }
+      
+      @include media('lg-up') {
+        margin-bottom: 4.625rem;
+      }
+    }
+    
+    h2 {
+      @extend .text-subheader-alt;
+      margin-bottom: 0.75rem;
+      color: color('black');
+
+      @include media('md-up') {
+        margin-bottom: 1.5rem;
+      }
+    }
+    
+    h3 {
+      @extend .text-subheader;
+      margin-bottom: 0.75rem;
+      color: color('black');
+
+      @include media('md-up') {
+        margin-bottom: 1.5rem;
+      }
+    }
+
+    p {
+      // internal links
+      a {
+        color: color('blue'); 
+        text-decoration: underline;
+
+        &:hover {
+          color: color('blue-darker'); 
+          text-decoration: none;
+        }
+      }
+
+      // both images and captions
+      > img, > img + em {
+        @extend .col-12;
+        display: block;
+
+        @include media('lg-up') {
+          width: 150%; // full width image on large viewports
+          position: relative;
+          left: 50%;
+          transform: translateX(-50%);
+        }
+      }
+
+      // image
+      > img {
+        @include media('md-up') {
+          margin-top: 1.5rem;
+        }
+
+        @include media('lg-up') {
+          // margin-top: 1.875rem;    
+          margin-top: 0;    
+        }
+      }
+
+      // caption
+      > img + em {
+        margin-top: 0.75rem;
+        @extend .text-body-sm;
+        font-style: normal;
+        color: color('gray');
+
+        @include media('md-up') {
+          margin-bottom: 4.5rem;
+        }
+        
+        @include media('lg-up') {
+          margin-bottom: 0;
+        }
+      }
+
+      & + ul, & + ol {
+        @include media('lg-up') {
+          margin-top: -3.125rem; // reducing margin between paragraphs & sibling lists
+        }
+      }
+    }
+
+    // paragraph with floating callout header
+    blockquote {
+      margin-top: 3.125rem;
+
+      @include media('lg-up') {
+        margin-top: 0;
+      }
+
+      // floating callout header
+      > h2 {
+        float: left;
+        width: calc(50% + 1.5rem);
+        padding-right: 1.5rem;
+        margin-bottom: 0;
+        transform: translateY(-0.25rem);
+
+        @include media('md-up') {
+          width: 50%;
+          margin-bottom: 1.875rem;
+          transform: translateY(0.75rem);
+        }
+
+        @include media('lg-up') {
+          width: 51%;
+          margin-left: calc(-8.3333333333% - 3rem);
+          padding-top: 0.75rem;
+          padding-bottom: 1.875rem;
+          padding-right: 2.25rem;
+          margin-bottom: 0;
+          transform: translateY(0);
+        }
+      }
+    }
+
+    .callout-card {
+      margin-bottom: 1.5rem;
```

Comment:
nit: looks like the callout card could use a bit more margin-bottom here. maybe increase it to 2.5? 

designs: 
<img width="373" alt="Screen Shot 2022-05-04 at 4 18 19 PM" src="https://user-images.githubusercontent.com/72356175/166819581-4d755920-90bf-4035-b28c-987fc68aaa8b.png">

staging link:
<img width="356" alt="Screen Shot 2022-05-04 at 4 18 32 PM" src="https://user-images.githubusercontent.com/72356175/166819583-15c589f1-acc1-46fd-9d4c-955d0ad9e08d.png">